editor/src/messages/portfolio/document/document_message_handler.rs

```diff
@@ -418,6 +436,7 @@ impl MessageHandler<DocumentMessage, (u64, &InputPreprocessorMessageHandler, &Pe
 				responses.add(DocumentOperation::ClearBlobURL { path: layer_path.into() });
 			}
 			GroupSelectedLayers => {
+				// TODO: Add code that changes the insert index of the new folder based on the selected layer
```

Comment:
todo still needs to be done?

Reply:
Still needs to be done. Started it but never finished. I haven't had a chance to work on it.